Formula/libbpf.rb

```diff
@@ -0,0 +1,39 @@
+class Libbpf < Formula
+  desc "Automated upstream mirror for libbpf stand-alone build"
+  homepage "https://github.com/libbpf/libbpf"
+  url "https://github.com/libbpf/libbpf/archive/refs/tags/v0.8.1.tar.gz"
+  sha256 "7bda8187efc619d1eb20a1ba5ab949dd68d40dd44945310c91ac0f915fa4a42b"
+  license "BSD-2-Clause"
+
+  depends_on "linux-headers@5.16" => :test
+  depends_on "elfutils"
+  depends_on :linux
+  depends_on "pkg-config"
+  depends_on "zlib"
+
+  def install
+    chdir "src" do
+      system "make"
+      system "make", "install", "PREFIX=#{prefix}"
```

Comment:
```suggestion
      system "make", "install", "PREFIX=#{prefix}", "LIBSUBDIR=lib"
```
This should prevent the Makefile from installing to `lib64`, which is not allowed in Homebrew.  That will save you the trouble of trying to link to `lib64` in the test.

Reply:
Thanks, I had just noticed this is well after investigating the libbpf Makefile a bit closer.